tests/cypress/integration/actions_tasks_objects/case_24_delete_unlock_lock_object.js

```diff
@@ -0,0 +1,129 @@
+// Copyright (C) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+import { taskName, labelName } from '../../support/const';
+
+context('Delete unlock/lock object', () => {
+    const caseId = '24';
+
+    const createRectangleShape2Points = {
+        points: 'By 2 Points',
+        type: 'Shape',
+        labelName: labelName,
+        firstX: 100,
+        firstY: 100,
+        secondX: 300,
+        secondY: 300,
+    };
+
+    function lockObject() {
+        cy.get('.cvat-objects-sidebar-state-item').within(() => {
+            cy.get('.cvat-object-item-button-lock').click();
+        });
+    };
+
+    function deleteObjectViaShortcut(shortcut, stateLockObject) {
+        if (stateLockObject == 'unlock') {
+            cy.get('.cvat-canvas-container').within(() => {
+                cy.get('.cvat_canvas_shape')
+                    .trigger('mousemove')
+                    .should('have.class', 'cvat_canvas_shape_activated');
+            });
+        };
+        cy.get('body')
+            .type(shortcut);
+    };
+
+    function clickRemoveOnDropdownMenu() {
+        cy.get('.ant-dropdown')
+            .not('.ant-dropdown-hidden')
+            .contains(new RegExp('^Remove$', 'g'))
+            .click({ force: true });
+    };
+
+    function deleteObjectViaGUIFromSidebar() {
+        cy.get('.cvat-objects-sidebar-states-list').within(() => {
+            cy.get('.cvat-objects-sidebar-state-item').within(() => {
+                cy.get('[aria-label="icon: more"]').click();
+            });
+        });
+        clickRemoveOnDropdownMenu();
+    };
+
+    function deleteObjectViaGUIFromObject() {
+        cy.get('.cvat-canvas-container').within(() => {
+            cy.get('.cvat_canvas_shape')
+                .trigger('mousemove')
+                .rightclick();
+        });
+        cy.get('.cvat-canvas-context-menu').within(() => {
+            cy.get('.cvat-objects-sidebar-state-item').within(() => {
+                cy.get('[aria-label="icon: more"]').click();
+            });
+        });
+        clickRemoveOnDropdownMenu();
+    };
+
+    function actionOnConfirmWindow(textBuntton) {
+        cy.get('.ant-modal-confirm').within(() => {
```

Comment:
The same comment.
Please, avoid using ``ant-*`` classes because they can be changed in other antd versions.

Reply:
done